packages/components/button/src/ButtonGroup/README.mdx

```diff
@@ -5,61 +5,44 @@ status: 'stable'
 section: 'buttonComponents'
 slug: /components/button-group/
 github: 'https://github.com/contentful/forma-36/tree/forma-v4/packages/components/button/src/ButtonGroup'
-storybook: 'https://v4-f36-storybook.netlify.app/?path=/story/components-button-buttongroup--basic'
 typescript: ./ButtonGroup.tsx
 ---
 
-import { Props } from '@contentful/f36-docs-utils';
-import { Button } from '@contentful/f36-button';
-import { ChevronDownIcon, PlusIcon } from '@contentful/f36-icons';
+`ButtonGroup` should be used to group buttons whose actions are related, with an option to collapse them together or split them with an equal amount of free space.
 
-## Code examples
+## Import
 
-Example of usage:
+```js static=true
+import { ButtonGroup } from '@contentful/f36-components';
+```
 
-```jsx
-// import { ButtonGroup, Button } from '@contentful/f36-button';
+## Examples
 
-<ButtonGroup testId="group-button">
-  <Button variant="secondary">Button</Button>
-  <Button variant="secondary">Button</Button>
-  <Button
-    variant="positive"
-    aria-label="Open dropdown"
-    startIcon={<ChevronDownIcon />}
-  />
-</ButtonGroup>
-```
+### Collapsed
 
-It's possible to add a divider for buttons that don't have border:
+The most common example is to group primary button with a secondary button or with a button that opens a dropdown with secondary actions.
 
-```jsx
-// import { ButtonGroup, Button } from '@contentful/f36-button';
-// import { ChevronDownIcon, PlusIcon } from '@contentful/f36-icons';
+Also, it is possible to add a divider for buttons that don't have border.
+
+```jsx file=../../examples/ButtonGroupCollapsedExample.tsx
 
-<ButtonGroup testId="group-button" withDivider>
-  <Button startIcon={<PlusIcon />} variant="positive">
-    Button
-  </Button>
-  <Button variant="positive" startIcon={<ChevronDownIcon />} />
-</ButtonGroup>
 ```
 
-Example of Group Button with spaced variant
+### Spaced
+
+Instead of collapsing buttons all together you can use `variant="spaced"` to make them spaced and control an empty space with `spacing` property.
```

Comment:
hmm, I don't get why we say collapsing buttons. but maybe it is my English hehehe. I always thought collapsing might be relevant to something like an accordion, menu, something that opens and closes.

Reply:
We could say just grouped (without space, if we want to be very specific)? dunno. maybe it's just me.

Reply:
Hmm... I'm questioning it now too, but I guess it's too late to change the name before the release. The better name would be `merged` or something like this. WDYT?

Reply:
yeah, that sounds better to me. Collapsed... hmmm. why we called the prop collapsed? 😄

Reply:
I have no idea 😢 but I think it's too late to make a breaking change. Should we create a new variant `merged` and keep `collapsed` as a synonym of `merged` for backward compatibility? 

We can remove `collapsed` in v5.